adr/recipe.py

```diff
@@ -49,7 +50,9 @@ def get_recipe_contexts(recipe, mod=None):
     run_context_def = context.get_context_definitions(run_contexts, specific_contexts)
     recipe_context_def.update(run_context_def)
 
-    return recipe_context_def
+    result = collections.OrderedDict()
+    result = context.sort_context_dict(recipe_context_def)
```

Comment:
Same comment as before, this second assignment removes the OrderedDict.

Reply:
The function `context.sort_context_dict()` returns an OrderedDict, so I think it will remain an OrderedDict. I tried to find some documents about OrderedDict assignment but couldn't find anything, can you help me to confirm this? Thanks!

Reply:
There's nothing special about assigning OrderedDicts, I'm just saying that first line is a no-op. You're assigning an OrderedDict() instance to `result`, then on the next line assigning a different and unrelated OrderedDict instance to result. So that first line has no purpose.

Reply:
Oh thanks I get it now. Just updated the code!